### ISSUE TITLE
change location programmatically?

### ISSUE THREAD

**Issue by DanielAndroidDev** [haktan313/H3DPathAIPro#1]
I would like to know if I can spawn an npc and give a different location to each one, or on the contrary I would like it to follow the player, (like the basic Ai Move to and changing the destination for target actor).

**Comment by haktan313** [comment]
Yes you can give each npc different position. also for now you can't use it with behaviour tree tasks like move to but I will do that too in the next times. for now unfortunately you can only give vector position.

**Comment by haktan313** [comment]
Yes you can give each npc different position. also for now you can't use it with behaviour tree tasks like move to but I will do that too in the next times. Follow me, when I made it will can see that. for now unfortunately you can only give vector position.

**Comment by DanielAndroidDev** [comment]
How can i do that?
The component HMoveTo dont let my change destination or set actor destination or im blind and dont see it

**Comment by haktan313** [comment]
I haven't tried such situations, but I guess by creating vector variables and setting them to the component target position when your npc reaches the previous position. For example, initially give position A to the target location of the component, and when it reaches it, set the other variable, position B. However, you may not be able to do this in blueprint, it can be done in C++. I will set it and upload it again

**Comment by DanielAndroidDev** [comment]
I forgot to start by saying: I just looked in blueprint and did not see the option.
If you could add it I would appreciate it.
I take this opportunity to say: could you create the component for version 5.4? I would be very grateful, (my c++ knowledge is basically zero T_T).

**Comment by haktan313** [comment]
I did the first thing you said, but I couldn't do the 5.4 upgrade for now.

**Comment by DanielAndroidDev** [comment]
Do you know an estimated date when you will be able to upgrade to version 5.4?

**Comment by Psygnomi** [comment]
Any update on 5.4 progress? Thank you :)

**Comment by haktan313** [comment]
i was working on other projects but i just started working on 3dpathfinding again. i am combining it with my HAIPro plugin, with this you can use the behaviour tree and in the behaviour tree you will have another move to task for it and i will also update it to the latest versions. if i can
